src/TopBar.js

```diff
@@ -15,11 +15,14 @@ export default () => {
         <Item to="/incidents" label="Incidents" />
         <Item to="/engagements" label="Engagements" />
         <div style={{ flexGrow: 1 }} />
+        <Button variant="dark" label="Profile" style={{ color: "black" }}  > {/* changing font color to dark doesn't work here for some reason, so I changed the button from light themed to dark themed for now */}
```

Comment:
Currently, the user needs to click the text to navigate to the profile page. Since this is a button, it should work to click anywhere inside the outline. Check out this example for how to use the onClick callback to navigate with history.push('/profile'): https://reacttraining.com/react-router/web/api/Hooks/usehistory

Reply:
Added a handleclick function to this button... but haven't tested this out yet. May I get a password reset? (Doh!)

Reply:
works!